source_code/aux_mcu_v4/src/COMMS/comms_aux_mcu_defines.h

```diff
@@ -14,5 +14,20 @@
 // HID payload size
 #define HID_PAYLOAD_SIZE            64
 
+//FIDO2 related lengths/sizes
+#define FIDO2_RPID_LEN 252
+#define FIDO2_USER_ID_LEN 65
+#define FIDO2_USER_NAME_LEN 65
+#define FIDO2_DISPLAY_NAME_LEN 32
+#define FIDO2_CLIENT_DATA_HASH_LEN 32
+#define FIDO2_RPID_HASH_LEN 32
+#define FIDO2_TAG_LEN 32
```

Comment:
hmmm.... are you sure?

this is in your struct:

uint8_t tag[FIDO2_TAG_LEN];                   //16

Reply:
Fixed comment